it/src/main/java/org/corfudb/universe/node/server/AbstractCorfuServer.java

```diff
@@ -0,0 +1,82 @@
+package org.corfudb.universe.node.server;
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang.StringUtils;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.corfudb.universe.node.NodeException;
+import org.corfudb.universe.universe.UniverseException;
+import org.corfudb.universe.universe.UniverseParams;
+
+import java.io.FileReader;
+import java.io.IOException;
+
+@Slf4j
+public abstract class AbstractCorfuServer<T extends CorfuServerParams, U extends UniverseParams> implements CorfuServer {
+
+    @Getter
+    @NonNull
+    protected final T params;
+    @NonNull
+    @Getter
+    protected final U universeParams;
+
+    protected AbstractCorfuServer(T params, U universeParams) {
+        this.params = params;
+        this.universeParams = universeParams;
+    }
+
+
+    /**
+     * This method creates a command line string for starting Corfu server
+     *
+     * @return command line parameters
+     */
+    protected String getCommandLineParams() {
+        StringBuilder cmd = new StringBuilder();
+        cmd.append("-a ").append(getNetworkInterface());
+
+        switch (params.getPersistence()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)